docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     build:
       context: .
       dockerfile: ./designer/Dockerfile
+      cache_from:
```

Comment:
This is in place to fix the pipeline cache when it fills up and causes a out of memory failure